cli/src/main/java/org/apache/servicecomb/toolkit/cli/ToolkitMain.java

```diff
@@ -33,7 +33,7 @@ public static void main(String[] args) {
 
     initialProjectVersion();
 
-    Cli.CliBuilder<Runnable> builder = Cli.<Runnable>builder("java -jar toolkit-cli-" + projectVersion + ".jar");
+    Cli.CliBuilder<Runnable> builder = Cli.<Runnable>builder("cli.sh");
```

Comment:
Not good, it's bind to linux

Reply:
Yes, it's not good. I will try to fix it

Reply:
I use system properties -Dscript.name to decide the name of the script
What do you think about it?